src/app/shared/components/user-card/user-card.component.ts

```diff
@@ -25,11 +25,12 @@ export class UserCardComponent implements OnInit, OnChanges {
   }
 
   ngOnChanges() {
+    // XXX: When is this called??
     this.fetchLearningObjects();
   }
```

Comment:
ngOnChanges is a lifecycle hook. Here is an explanation: https://v17.angular.io/guide/lifecycle-hooks